code/modules/mob/living/living.dm

```diff
@@ -205,30 +205,32 @@
 	if(!client && (mob_size < MOB_SIZE_SMALL))
 		return
 	now_pushing = TRUE
-	var/t = get_dir(src, AM)
+	var/dir_to_target = get_dir(src, AM)
+	if(!dir_to_target)
+		dir_to_target = dir
```

Comment:
What happen here?

Reply:
I've added a comment to explain the scenario where this happens. Feel free to check it again in light of that.